packages/router/src/Set.tsx

```diff
@@ -44,7 +44,7 @@ export function Set<WrapperProps>(props: SetProps<WrapperProps>) {
     private: privateSet,
     unauthenticated,
     roles,
-    whileLoadingAuth,
+    whileLoadingAuth: WhileLoadingAuth,
```

Comment:
These things always confuse me 😕 Could you do this before? Can you still do it? `<Set whileLoadingAuth={() => <LoadingSpinner />}>`

Reply:
I won't pretend I totally understand it either at the moment, but I think that if we don't use angle brackets like `<whileLoadingAuth />` (and the convention's to capitalize it so that's why I renamed it) Babel won't turn it into `React.createElement(whileLoadingAuth)`. And I think that this situation isn't like `createCell` which returns a React Component. Instead users are passing components to `whileLoadingAuth`, and it's the router's job to render them as elements.